src/components/layout/layout.js

```diff
@@ -137,6 +152,8 @@ const Layout = ({ children, type, interim_type, padding_top, no_login_signup })
 Layout.propTypes = {
     children: PropTypes.node.isRequired,
     interim_type: PropTypes.string,
+    is_p2p_v1: PropTypes.bool,
```

Comment:
i think this is too specific, can we name it to navigation_type instead?

Reply:
Alright, will refactor it to a more general usage.